content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -77,46 +93,69 @@ Once the credit note has been posted, a :guilabel:`Credit Notes` smart button wi
 
-The :guilabel:`Coupon Code` can be copied directly from the pop-up window (by clicking the
-:guilabel:`Copy` button), or sent in an email by clicking :guilabel:`Send`.
+.. note::
+   To create a new :guilabel:`Coupon Program`, navigate to :menuselection:`Sales app --> Products
+   --> Discount & Loyalty` and click :guilabel:`New`. To make the program available to share with
+   *Helpdesk* customers, the :guilabel:`Program Type` **must** be set to :guilabel:`Coupons`. This
+   generates single-use coupon codes that grant immediate access to rewards and discounts.
+
+  Coupon programs can also be created in the *Point of Sale* application or *Website* application.
+  Refer to :doc:`discount and loyalty programs
+  </applications/sales/sales/products_prices/loyalty_discount>` for more information.
+
+Click on the :guilabel:`Valid Until` field, and use the pop-up calendar to select an expiration date
+for this coupon code. If this field is left blank, the code does **not** expire.
+
+Click :guilabel:`Send by Email` to compose an email to send to the customer with the coupon code.
 
 .. note::
-   When emailing a coupon code, all the followers of the ticket will be added as recipients to the
-   email. Additional recipients can be added to the email as well, in the :guilabel:`Recipients`
-   field of the :guilabel:`Compose Email` pop-up window.
+   When emailing a coupon code, **all** the followers of the ticket are added as recipients to
+   the email. Additional recipients can be added to the email as well, in the :guilabel:`Recipients`
+   field of the :guilabel:`Compose Email` pop-up window. If an expiration date was selected for the
+   code, it is included in the message template.
 
    .. image:: after_sales/after-sales-coupon-email.png
       :align: center
       :alt: View of an email draft window with coupon code.
 
-Once a :guilabel:`Coupon Code` has been generated, a :guilabel:`Coupons` smart button will be added
-to the top of the ticket; click the smart button to view the coupon code, expiration date, and
+Click :guilabel:`Get Share Link` to generate a link to send directly to the customer. Doing so opens
+a :guilabel:`Share Coupons` pop-up window. Click the :guilabel:`Copy` button next to the
+:guilabel:`Share Link` field and paste the results to any communication with the customer. When the
+customer uses the link, the code is automatically applied to their cart.
+
+After a :guilabel:`Coupon Code` has been generated, a :guilabel:`Coupons` smart button is added to
+the top of the ticket; click the smart button to view the coupon code, expiration date, and
 additional information.
 
 .. image:: after_sales/after-sales-coupon-smart-button.png
    :align: center
    :alt: View of the smart buttons on a ticket focusing on the coupon button.
 
 .. seealso::
-   `Coupons <https://www.odoo.com/slides/slide/coupon-programs-640?fullscreen=1>`_
+   - `Coupons <https://www.odoo.com/slides/slide/coupon-programs-640?fullscreen=1>`_
```

Comment:
This URL does not seem to be valid, can you please check this?
<img width="619" alt="Screenshot 2024-03-05 at 10 22 20 AM" src="https://github.com/odoo/documentation/assets/36018073/2d36fcd8-fde2-46da-b1a7-9fdcbeced2e5">